web/src/routes/(user)/albums/[albumId]/+page.svelte

```diff
@@ -220,12 +220,12 @@
 
   onMount(async () => {
     if (album.sharedUsers.length > 0) {
-      getFavorite();
-      getNumberOfComments();
+      await getFavorite();
+      await getNumberOfComments();
```

Comment:
This now runs sequentially instead of in parallel, but it probably doesn't matter that much

Reply:
updated